### ISSUE TITLE
Enum metric type not supported

### ISSUE THREAD

**Issue by Jean-PhilippeD** [rycus86/prometheus_flask_exporter#163]
Hello,

It seems [Enum ](https://github.com/prometheus/client_python#enum) is not supported, or I'm using it the wrong way ?

**Comment by rycus86** [comment]
No, you're not wrong. I haven't seen that before, sounds like we should add support.
I'd be happy to review a PR for this if you'd like to attempt it?

**Comment by Jean-PhilippeD** [comment]
Hi,
I think I will not be able to make a PR, I'd like but I lack of time for the moment and I'm new to flask_prometheus_exporter so I would need more times to understand how it works. 
Maybe you can help me to understand how metric from python client works within the flask exporter..

For now I think I will use a gauge and map a numeric value to a string value.
